src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -72,7 +79,9 @@ def Db2Lin(x):
   return 10**(x / 10.)
 
 def Lin2Db(x):
-  return 10 * np.log10(x)
+  if not np.isscalar(x):
+    return 10 * np.log10(x.clip(min=1e-100))
```

Comment:
could you check that this line work correctly when x is a list and not an np array?

Reply:
I dont think we use this Lin2Db for a list in this file, and this function is not designed to work with any Python sequence (list, dict, set,), only with  numerical container: (scalar and ndarray) as used in the module. 
But since it is easy to extend it, please check the new form.